### DISCUSSION TITLE
Realistic Expectations

### DISCUSSION THREAD

**Discussion by bwthor** [QuantumEntangledAndy/neolink#158] [General]
I have four B800 cameras.  If I'm running neolink on the same computer hosting Blue Iris, can I expect that to handle things OK?  i7 7700 and 32GB RAM.
4k main stream at 10fps
If not, should I consider running neolink on another computer?

**Comment by QuantumEntangledAndy** [comment]
Depends on the number of cameras and other network demands. I run mine on an Rpi3B so its possible but I also run mine with all the on pause options since I don't need it recording all the time. Best recomendation if to just test it. Perhaps you can report back with your results do others can learn from it to

**Reply by bwthor** [reply]
Thanks, I have been testing some, and getting mixed results.  I still have 6 other 1080p cameras running in production, while testing these four.  I will report back with results, but it will be a little while before I can get things all set properly.

**Reply by QuantumEntangledAndy** [reply]
I see we are currently tracking some unbound memory growth over extended run periods (hopefully will be all fixed on the next release). So bare that in mind while testing, if things start to break because ram is getting too large

**Reply by bwthor** [reply]
Oh...I forgot to thank you for picking up this project and sharing it with the rest of us!
